contribs/src/main/java/com/netflix/conductor/contribs/kafka/workers/KafkaStreamsWorkersObservableQueue.java

```diff
@@ -29,8 +29,9 @@
 public class KafkaStreamsWorkersObservableQueue implements ObservableQueue, Runnable {
     private static final Logger logger = LoggerFactory.getLogger(KafkaStreamsWorkersObservableQueue.class);
     private static final int KEY_ERROR_BRANCH = 0;
-    private static final int REGISTER_BRANCH = 1;
-    private static final int VALUE_ERROR_BRANCH = 2;
+    private static final int VALUE_ERROR_BRANCH = 1;
+    private static final int UNIQUE_URI_ERROR_BRANCH = 2;
+    private static final int REGISTER_BRANCH = 3;
     private static final int REGISTRATION_SUCCESS_BRANCH = 0;
     private static final int REGISTRATION_UNSUCCESSFUL_BRANCH = 1;
```

Comment:
This whole section should be configurable

Reply:
No, this part should not because this is something specific to a process of that class kafka streams